### ISSUE TITLE
Autoplay timeout isn't reset when changing a slide on mobile

### ISSUE THREAD

**Issue by escanxr** [antoniandre/vueper-slides#131]
Hi!

I noticed that when Vueper Slides is used on a mobile (Chrome + Android in my case), the autoplay timeout is not reset after a slide change.

If you activate the autoplay with a duration of 7000 for example, and you wait 3 seconds before manually going to the next slide, then this slide will go to the next after 4 seconds instead of 7.

The problem also appears when using Chrome's mobile simulator, but not when simply resizing the window.

This is my code:

```javascript
  <vueper-slides
    autoplay
    touchable
    :slide-ratio="slideRatio"
    :visible-slides="visibleSlides"
    :gap="2"
    slide-multiple
    :arrows="false"
    bullets-outside
    class="no-shadow"
    duration="7000"
    :dragging-distance="70"
  >
      <vueper-slide v-for="item in items" :key="item.url" :link="item.url" open-in-new>
        <template v-slot:content>
          <slot v-bind:item="item" />
        </template>
      </vueper-slide>
  </vueper-slides>
```

Am I missing something?
If it's a bug, I'll try to take a look at it in the next few days.

**Comment by antoniandre** [comment]
Sorry to be late to the party,

Thank you @escanxr for reporting, and @roenfeldt for narrowing down and providing a potential solution.
I will have a look what I can do, but I agree it should be addressed internally. :)

**Comment by antoniandre** [comment]
Hi @escanxr, @roenfeldt, this should now be fixed in version 2.15.0.

**FIXES:**
- Touch device or not, pause the autoplay while dragging (you may hold the slide).
- Touch device or not, any interaction while autoplaying will reset the timer to the next slide.

**Also added:**
- I also added the `pauseOnTouch` option (when `autoplay` is true) and set it to true by default.
   When set to true on touch device, any interaction with the slideshow will pause it, and only a click outside will resume the autoplay.
- If you set the `pauseOnTouch` option to false, any interaction with the slideshow will not pause it.

Hope it helps! :)
Let me know if you find anything not working as expected.
